bridge/engine/AudioForwarderRewriteAndSendJob.cpp

```diff
@@ -1,19 +1,54 @@
 #include "bridge/engine/AudioForwarderRewriteAndSendJob.h"
+#include "bridge/engine/SsrcInboundContext.h"
 #include "bridge/engine/SsrcOutboundContext.h"
 #include "codec/Opus.h"
 #include "rtp/RtpHeader.h"
 #include "transport/Transport.h"
 #include "utils/OutboundSequenceNumber.h"
 
+namespace
+{
+
+inline void rewriteHeaderExtensions(rtp::RtpHeader* rtpHeader,
+    const bridge::SsrcInboundContext& senderInboundContext,
+    const bridge::SsrcOutboundContext& receiverOutboundContext)
+{
+    assert(rtpHeader);
+
+    const auto headerExtensions = rtpHeader->getExtensionHeader();
+    if (!headerExtensions)
+    {
+        return;
+    }
+
+    for (auto& rtpHeaderExtension : headerExtensions->extensions())
+    {
+        if (senderInboundContext._rtpMap._audioLevelExtId.isSet() &&
+            rtpHeaderExtension.getId() == senderInboundContext._rtpMap._audioLevelExtId.get())
+        {
+            rtpHeaderExtension.setId(senderInboundContext._rtpMap._audioLevelExtId.get());
```

Comment:
We want to set it to outbound context audio level ext id, right?

Reply:
Of course, thanks!